osquery/events/tests/producer.cpp

```diff
@@ -8,7 +8,10 @@
  *
```

Comment:
What about just moving them to `osquery/events/kernel/tests/`  kind of the way the other test directories are like `osquery/events/linux/tests`

Also worth noting they are going to disappear soon.